Makefile

```diff
@@ -176,6 +176,13 @@ out/minikube_$(DEB_VERSION).deb: out/minikube-linux-amd64
 	dpkg-deb --build out/minikube_$(DEB_VERSION)
 	rm -rf out/minikube_$(DEB_VERSION)
 
+out/minikube_$(DEB_VERSION).tar.gz: out/localkube out/minikube-linux-amd64 out/minikube-darwin-amd64 out/minikube-windows-amd64.exe
```

Comment:
I think its ok to include the kvm and xhyve drivers even for the virtualbox users.  They're pretty small binaries.  The installation script (if/however it exists) can always ask which one.

Platform-driver might be a little unwieldy 

linux-vbox
linux-kvm
linux-none

mac-vbox
mac-xhyve
mac-vmwarefusion

windows-vbox
windows-hyperv